Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -360,11 +362,20 @@ def bump_formula_pr
       if github_release_data.present?
         pre = "pre" if github_release_data["prerelease"].present?
         pr_message += <<~XML
+
           <details>
-            <summary>#{pre}release notes</summary>
-            <pre>#{github_release_data["body"]}</pre>
```

Comment:
The `<pre>` [is intentional](https://github.com/Homebrew/brew/pull/14978) otherwise it's super noisy and the release notes linking to other PRs get expanded wrongly.

Reply:
And, another thought about the commit message release notes references: they'll make the issues they reference in projects super noisy since every reference in a commit message is then linked back to the issue it references (see https://github.com/Homebrew/brew/issues/14717 for an example of _legitimate_ uses of this cluttering up the log) - that could be annoying for project maintainers seeing multiple of these "pushed a commit that referenced this issue" for a) a person's fork and b) our push to master and c) any force-pushes in between.

Reply:
How about wrapping only the issue references in backticks, instead of the whole thing?